benchcab/benchcab.py

```diff
@@ -192,7 +192,24 @@ def checkout(self):
     def build(self):
         """Endpoint for `benchcab build`."""
         for repo in self.repos:
-            repo.build(modules=self.config["modules"], verbose=self.args.verbose)
+            if repo.build_script:
+                print(
+                    "Compiling CABLE using custom build script for "
+                    f"realisation {repo.name}..."
+                )
+                repo.custom_build(
+                    modules=self.config["modules"], verbose=self.args.verbose
+                )
+            else:
+                print(
+                    f"Compiling CABLE {'with MPI' if internal.MPI else 'serially'} for "
```

Comment:
This may be semantics, but you don't often see conditionals embedded directly into an f-string, rather, the conditional bits are evaluated prior then subbed in place for clarity.